singer-io/tap-jira
# TDL-14738 Update primary key for issue_transition stream

# Description of change
- Added the `issue_id` key in the primary key of the `issue_transition` stream.
- Added assertion to check unique records by primary key in pagination test case.

# Manual QA steps
 - Run the sync mode and verify that all records are unique by the composite primary key of `id` and `issue_id`.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
